mail/inc/class.mail_ui.inc.php

```diff
@@ -1478,6 +1478,16 @@ private function get_actions()
 				//'onExecute' => 'javaScript:app.mail.mail_dragStart',
 			)
 		);
+		//hook, mail extra actions
```

Comment:
Adding actions can also be done in the etemplate2_before_exec hook.

Sorry that I forgot to mention that before.

Only the mail_compose_after_save is necessary.

Ralf